roles/tm-deepsec-aws-setup/templates/silent-install-tm.conf.j2

```diff
@@ -0,0 +1,34 @@
+AddressAndPortsScreen.ManagerAddress=10.0.2.197
```

Comment:
you can get this from the hostvars, nodepool keys. Look into the inventory file when the a job finished to see the stanza.